src/tmx_json.c

```diff
@@ -205,39 +207,164 @@ static int pjson_tile(json_t *tile_el, tmx_tile **tile_headaddr) {
 	return 1;
 }
 
+
+/* This is a rather slow solution - anyone with more spare time, feel free to rework the whole structure of
+   the tiles list to make this faster */

```

Comment:
Create a branch from your master, and start to work on the branch:
`git branch new-branch-name` to create a branch named 'new-branch-name', starting from the current commit.
`git checkout new-branch-name` to go to the newly created branch
`git checkout master` to go back to the branch 'master'.

Because your pull request is on the branch master, github will add new commits on the branch master, but won't add commits from another branch.